test/test-network/systemd-networkd-tests.py

```diff
@@ -58,260 +53,546 @@
 ubsan_options=None
 with_coverage=False
 
-running_units = []
+active_units = []
+protected_links = {
+    'erspan0',
+    'gre0',
+    'gretap0',
+    'ifb0',
+    'ifb1',
+    'ip6_vti0',
+    'ip6gre0',
+    'ip6tnl0',
+    'ip_vti0',
+    'lo',
+    'sit0',
+    'tunl0',
+}
+saved_routes=None
+saved_ipv4_rules=None
+saved_ipv6_rules=None
+
+def rm_f(path):
+    if os.path.exists(path):
+        os.remove(path)
+
+def rm_rf(path):
+    shutil.rmtree(path, ignore_errors=True)
+
+def cp(src, dst):
+    shutil.copy(src, dst)
+
+def cp_r(src, dst):
+    shutil.copytree(src, dst, copy_function=shutil.copy)
+
+def mkdir_p(path):
+    os.makedirs(path, exist_ok=True)
+
+def touch(path):
+    Path(path).touch()
+
+def check_output(*command, text=True, **kwargs):
+    # This checks the result and returns stdout on success.
+    command = command[0].split() + list(command[1:])
+    return subprocess.run(command, check=True, universal_newlines=text, capture_output=True, **kwargs).stdout.rstrip()
```

Comment:
Looks like `capture_output` was introduced in Python 3.7, and the C8S nodes in CentOS CI are on Python 3.6. Should I bump it to Python 3.8 or using `stdout=PIPE` and `stderr=PIPE` would be a sufficient alternative?

Reply:
Thank you for catching that. That's not necessary, Now `std{out,err}=PIPE` are used.